### ISSUE TITLE
Actualización

### ISSUE THREAD

**Issue by dixonmartinez** [willicab/infove-api#1]
Hola, existe alguna actualización al respecto?

**Comment by AngelJRomero21** [comment]
Pudiste ver algo?

**Comment by dixonmartinez** [comment]
No, nada, este proyecto no le dieron continuidad por lo que veo

**Comment by AngelJRomero21** [comment]
Sería interesante retomar este proyecto, sin embargo actualmente hay muchas complicaciones con las apis y páginas gubernamentales.

Lo que se podría retomar en este caso sería lo de CANTV y las cédulas.
